apps/settings/templates/settings/personal/development.notice.php

```diff
@@ -30,43 +30,43 @@
 		<?php print_unescaped(str_replace(
 			[
 				'{facebookimage}',
-				'{twitterimage}',
+				'{ximage}',
 				'{mastodonimage}',
 				'{rssimage}',
 				'{mailimage}',
 				'{facebookopen}',
-				'{twitteropen}',
+				'{xopen}',
 				'{mastodonopen}',
 				'{rssopen}',
 				'{newsletteropen}',
 				'{linkclose}',
 				'{facebooktext}',
-				'{twittertext}',
+				'{xtext}',
 				'{mastodontext}',
 				'{rsstext}',
 				'{mailtext}',
 			],
 			[
 				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="m7 5c0-1.1 0.9-2 2-2h1.5v2h-1c-0.27 0-0.5 0.23-0.5 0.5v1h1.5v2h-1.5v4.5h-2v-4.5h-1.5v-2h1.5z" fill="var(--color-main-background)"/></svg>',
-				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="m9.784 4c-1.0945 0-1.982 0.89-1.982 1.982 0 0.167 0.026 0.336 0.06 0.474h-0.153c-1.361-0.077-2.577-0.56-3.445-1.512-0.19-0.204-0.368-0.422-0.518-0.654-0.294 0.509-0.325 1.1-0.19 1.648 0.123 0.41 0.34 0.776 0.64 1.05l0.15 0.1364-0.19-0.041c-0.245-0.068-0.49-0.15-0.722-0.245 0.027 0.49 0.218 0.94 0.517 1.294 0.286 0.34 0.667 0.6 1.103 0.722-0.31 0.032-0.6165 0.0236-0.926 0.0135 0.3132 0.763 1.035 1.322 1.8927 1.3897-0.79 0.585-1.77 0.912-2.82 0.912 0.7762 0.477 1.661 0.777 2.628 0.831h0.627c2.982-0.163 5.338-2.63 5.34-5.6526v-0.286c0.11-0.095 0.218-0.19 0.327-0.3 0.259-0.2316 0.49-0.504 0.681-0.79-0.313 0.204-0.681 0.3133-1.09 0.313h-0.014c0.366-0.2453 0.653-0.613 0.8-1.035-0.41 0.212-0.802 0.3433-1.23 0.425-0.46-0.4-0.83-0.675-1.486-0.675z" fill="var(--color-main-background)"/></svg>',
+				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="M 2.6425781 1.8 C 2.2543781 1.8 2.0276281 2.2362184 2.2488281 2.5558594 L 5.8417969 7.6900156 L 2.2347656 11.9074218 C 2.0013656 12.1798212 2.1947156 12.6 2.5535156 12.6 L 2.9519531 12.6 C 3.1279531 12.6 3.2949844 12.5228622 3.4089844 12.3890624 L 6.5519531 8.7065156 L 8.9179688 12.0878908 C 9.1339688 12.408984 9.5099768 12.6 10.0011718 12.6 L 11.9015628 12.6 C 12.2897628 12.6 12.5173128 12.163312 12.2953128 11.845312 L 8.5042968 6.4195156 L 11.9402344 2.3941408 C 12.139431 2.1601408 11.973215 1.8 11.6651601 1.8 L 11.1585938 1.8 C 10.9835938 1.8 10.816734 1.876453 10.702734 2.0197656 L 7.7982656 5.4058438 L 5.6367188 2.3132812 C 5.4145313 1.9913687 4.9541875 1.8 4.6535156 1.8 L 2.6425781 1.8 z" fill="var(--color-main-background)"/></svg>',
```

Comment:
@JuliaKirschenheuter @marcoambrosini any tips on how to properly generate the path in that it centralizes itself in the circle path?

Maybe a tool? Too bad I can't write in svg :grin:

Reply:
```suggestion
				'<svg aria-hidden="true" width="50" height="50" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="8" fill="var(--color-text-maxcontrast)"/><path d="m 3.384891,2.6 c -0.3882,0 -0.61495,0.4362184 -0.39375,0.7558594 L 6.5841098,8.4900156 2.9770785,12.707422 C 2.7436785,12.979821 2.9370285,13.4 3.2958285,13.4 H 3.694266 c 0.176,0 0.3430313,-0.07714 0.4570313,-0.210938 L 7.294266,9.5065156 9.6602817,12.887891 C 9.8762817,13.208984 10.25229,13.4 10.743485,13.4 h 1.900391 c 0.3882,0 0.61575,-0.436688 0.39375,-0.754688 L 9.2466097,7.2195156 12.682547,3.1941408 C 12.881744,2.9601408 12.715528,2.6 12.407473,2.6 h -0.506566 c -0.175,0 -0.34186,0.076453 -0.45586,0.2197656 L 8.5405785,6.2058438 6.3790317,3.1132812 C 6.1568442,2.7913687 5.6965004,2.6 5.3958285,2.6 Z" fill="var(--color-main-background)"/></svg>',
```
Centered it with Inkscape

Reply:
thanks @provokateurin looks way better.